css/index.css

```diff
@@ -1,3 +1,700 @@
+    margin-bottom: 10px;
+}
+
+/* DESKTOP */
+
+/* INDEX HOME PAGE */
+
+.container {
+  border: 2px solid red;
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+/* shared header */
+
+header {
+	display: flex;
+	justify-content: space-between;
+	
+}
+
+header nav {
+  /* border: 1px solid red; */
+  display: flex;
+  justify-content: space-between;
+  width: 550px;
+  margin: 0 50px;
+  margin-top: 35px;
+}
+
+header nav a {
+  font-size: 14px;
+  opacity: .6;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header .logo {
+  margin-top: 25px;
+  margin-right: 50px;
+}
+
+/* UNIQUE TO INDEX */
+/* cta */
+
+.cta {
+	display: flex;
+	/* border: 2px solid black; */
+	justify-content: center;
+	align-items: center;
+	height: 500px;
+	justify-content: space-evenly;
+}
+
+.cta .cta-text {
+	/* display: flex; */
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+  font-size: 77px;
+  text-align: center;
+}
+
+.cta .cta-text button {
+  background: white;
+  padding: 8px 44px;
+  font-size: 17px;
+  /* border: 1px solid black; */
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta .cta-text button {
+	align-self: flex-end;
+  }
+
+header .cta-img {
+    border: 2px solid red;
+    display: none;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  /* border: 2px solid red; */
+  display: flex;
+  justify-content: space-between;
+}
+
+.main-content .top-content .text-content {
+  width: 450px;
+}
+
+.main-content .top-content .text-content h4 {
+  font-size: 18px;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  justify-content: space-between;
+  
+  margin-top: 37px;
+  
+}
+
+.main-content .bottom-content .text-content {
+  width: 290px;
+  
+
+}
+
+.main-content .bottom-content .text-content h4 {
+	font-size: 18px;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact  h4 {
+	font-size: 18px;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+
+/* TABLET */
+
+@media (max-width: 800px) {
+    /* TABLET HEADER */
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+        
+        
+    }
+
+    header .logo{
+          display: flex;
+          justify-content: center;
+          padding-top: 2px;
+          padding-right: 27px;         
+      }
+      
+      header nav {
+        display: flex; 
+        
+        width: 710px;
+        margin-top: 40px;
+        margin-left: 25px;  
+      }
+
+      header nav a {
+        font-size: 16px;
+        opacity: .6;
+        color: gray;
+        text-decoration: none;
+      }
+      
+      /* TABLET CTA  */
+      .cta-img {
+          display: none;
+      }
+
+      .cta {
```

Comment:
I could see lot of inconsistency in the way you are formatting, make sure you maintain it well because readability is a priority when you are working in a team.